tfx_addons/example_filter/executor.py

```diff
@@ -0,0 +1,367 @@
+                if tfma.is_batched_input(eval_shared_model, eval_config):
+                    tfxio_factory = tfxio_utils.get_tfxio_factory_from_artifact(
+                        examples=input_dict[standard_component_specs.EXAMPLES_KEY],
+                        telemetry_descriptors=_TELEMETRY_DESCRIPTORS,
+                        schema=schema,
+                        raw_record_column_name=tfma_constants.ARROW_INPUT_COLUMN)
+                    # TODO(b/161935932): refactor after TFXIO supports multiple patterns.
+                    for split in example_splits:
+                        split_uris = artifact_utils.get_split_uris(
+                            input_dict[standard_component_specs.EXAMPLES_KEY], split)
+                        for index in range(len(split_uris)):
+                            split_uri = split_uris[index]
+                            file_pattern = io_utils.all_files_pattern(split_uri)
+                            tfxio = tfxio_factory(file_pattern)
+                            data = (
+                                    pipeline
+                                    | f'ReadFromTFRecordToArrow[{split}][{index}]' >>
+                                    tfxio.BeamSource())
+                            examples_list.append(data)
+                    if schema is not None:
+                        # Use last tfxio as TensorRepresentations and ArrowSchema are fixed.
+                        tensor_adapter_config = tensor_adapter.TensorAdapterConfig(
+                            arrow_schema=tfxio.ArrowSchema(),
+                            tensor_representations=tfxio.TensorRepresentations())
+                else:
+                    for split in example_splits:
+                        split_uris = artifact_utils.get_split_uris(
+                            input_dict[standard_component_specs.EXAMPLES_KEY], split)
+                        for index in range(len(split_uris)):
+                            split_uri = split_uris[index]
+                            file_pattern = io_utils.all_files_pattern(split_uri)
+                            data = (
+                                    pipeline
+                                    | f'ReadFromTFRecord[{split}][{index}]' >>
+                                    beam.io.ReadFromTFRecord(file_pattern=file_pattern))
+                            examples_list.append(data)
+
+                custom_extractors = udf_utils.try_get_fn(
+                    exec_properties=exec_properties, fn_name='custom_extractors')
+                extractors = None
+                if custom_extractors:
+                    extractors = custom_extractors(
+                        eval_shared_model=eval_shared_model,
+                        eval_config=eval_config,
+                        tensor_adapter_config=tensor_adapter_config)
+
+                try:
+                    examples_list = filter(input_dict[standard_component_specs.FILTER_FUNCTIONS_KEY][0], examples_list)
```

Comment:
This won't work. examples_list contains a few Beam pipelines not examples itself.